cmd/contractWatcher.go

```diff
@@ -91,7 +92,7 @@ func contractWatcher() {
 	ticker := time.NewTicker(5 * time.Second)
 	defer ticker.Stop()
 
-	blockChain := getBlockChain()
+	blockChain := getBlockChain(subCommand)
```

Comment:
Ugh I'm hesitant to pass values that are just logging metadata along through the application. I'd prefer storing this in a global variable somewhere that's set early by Cobra, that we can reference when logging.